apps/backend/cli/formatters.py

```
@@ -0,0 +1,253 @@
+from rich.panel import Panel
+from rich.table import Table
+from rich.text import Text
+
+from apps.backend.core.configuration.validation.base import (
+    ValidationResult,
+    ValidationStatus,
+)
+from apps.backend.core.configuration.validation.orchestrator import ValidationReport
+
+
+def format_text(report: ValidationReport, verbose: bool = False) -> str:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)